lib/matplotlib/tests/test_cm_stubs.py

```diff
@@ -0,0 +1,41 @@
+import matplotlib.pyplot as plt
+import re
+from pathlib import Path
+from matplotlib.colors import Colormap
+
+
+def test_cm_stub_matches_runtime_colormaps():
+    runtime_cm = plt.cm
+    runtime_cmaps = {
+        name
+        for name, value in vars(runtime_cm).items()
+        if isinstance(value, Colormap)
+    }
+
+    cm_pyi_path = Path(__file__).parent.parent / "cm.pyi"
+    assert cm_pyi_path.exists(), f"{cm_pyi_path} does not exist"
+
+    with cm_pyi_path.open("r", encoding="utf-8") as f:
+        pyi_content = f.read()
+
+    stubbed_cmaps = set(
+        re.findall(r"^(\w+):\s+colors\.Colormap", pyi_content, re.MULTILINE)
+    )
+
+    assert runtime_cmaps, (
+        "No colormaps variables found at runtime in matplotlib.colors"
+    )
+    assert stubbed_cmaps, (
+        "No colormaps found in cm.pyi"
+    )
+
+    missing_in_stub = runtime_cmaps - stubbed_cmaps
+    extra_in_stub = stubbed_cmaps - runtime_cmaps
+
+
+    assert not missing_in_stub, (
+        f"Colormaps missing in cm.pyi: {sorted(missing_in_stub)}"
+    )
+    assert not extra_in_stub, (
+        f"Extra colormaps in cm.pyi: {sorted(extra_in_stub)}"
+    )
```

Comment:
You can just do:
```suggestion
    assert runtime_cmaps == stubbed_cmaps
```
Pytest will print out the differing items for you.